apps/web/src/features/layout/components/ThemeEnvironment/forestEnvironment.module.css

```
@@ -0,0 +1,200 @@
+  border: 1px solid rgba(232, 197, 71, 0.28);
+  box-shadow: 0 10px 28px rgba(4, 10, 7, 0.28);
+  pointer-events: none;
+}
+
+.tooltip strong,
+.tooltip span {
+  display: block;
+}
+
+.tooltip strong {
+  font-family: "Bitter", serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Bitter" (font-family-name-quotes)

(font-family-name-quotes)